docs/README.md

```diff
@@ -8,6 +8,16 @@ Read on to learn more about viewing documentation in plain text (i.e., markdown)
 documentation yourself. Why build it yourself? So that you have the docs that corresponds to
 whichever version of Spark you currently have checked out of revision control.
 
+## Prerequisites
+The Spark documenation build uses a number of tools to build HTML docs and API docs in Scala, Python
+and R. To get started you can run the following commands
+
+    $ sudo gem install jekyll
```

Comment:
@rxin can probably add more, but I meant this to be redundant in the sense this section had a bunch of commands you could copy-paste while the succeeding sections had the rationale / breakdown of what was required when.